src/pricing/index.njk

```diff
@@ -580,10 +580,10 @@ hubspot:
     }
 
     // Setup the sliders
-    setupPricingSlider("#team-annual-slider", 250, 5, "#team-annual-price-int", "#team-annual-price-dec", "#team-annual-price-note", "#team-annual-value", true);
-    setupPricingSlider("#enterprise-annual-slider", 500, 10, "#enterprise-annual-price-int", "#enterprise-annual-price-dec", "#enterprise-annual-price-note", "#enterprise-annual-value", true);
-    setupPricingSlider("#team-monthly-slider", 300, 5, "#team-monthly-price-int", "#team-monthly-price-dec", "#team-monthly-price-note", "#team-monthly-value", false);
-    setupPricingSlider("#enterprise-monthly-slider", 600, 10, "#enterprise-monthly-price-int", "#enterprise-monthly-price-dec", "#enterprise-monthly-price-note", "#enterprise-monthly-value", false);
+    setupPricingSlider("#team-annual-slider", 250, 3, "#team-annual-price-int", "#team-annual-price-dec", "#team-annual-price-note", "#team-annual-value", true);
+    setupPricingSlider("#enterprise-annual-slider", 500, 5, "#enterprise-annual-price-int", "#enterprise-annual-price-dec", "#enterprise-annual-price-note", "#enterprise-annual-value", true);
+    setupPricingSlider("#team-monthly-slider", 300, 3, "#team-monthly-price-int", "#team-monthly-price-dec", "#team-monthly-price-note", "#team-monthly-value", false);
+    setupPricingSlider("#enterprise-monthly-slider", 600, 5, "#enterprise-monthly-price-int", "#enterprise-monthly-price-dec", "#enterprise-monthly-price-note", "#enterprise-monthly-value", false);
```

Comment:
I can see new minimum instances. There are other sections of the code that need to be updated as well, I'll update what's needed (maybe I'll iterate on the script instead)

Reply:
I already updated the script